docs/user/orion.rst

```diff
@@ -0,0 +1,92 @@
+
+General Usage
+^^^^^^^^^^^^^
+
+After installation, Orion provides a custom tool window at *View -> Tool Windows -> Artemis*. This window contains an integrated browser displaying Artemis.
+
+At the top of the tool window is a back button |back-button|. Clicking it will return to the initial page, that is:
+
+- The exercise details page if an exercise is opened as a student
+- The exercise edit in editor page if an exercise is opened as an instructor
+- The assessment dashboard if an exercise is opened as a tutor, or, if a submission is downloaded, the assessment details page of that submission
+- The Artemis home page if no Artemis project is opened
+
+Settings
+^^^^^^^^
+
+Orion's settings are at *Settings -> Tools -> Orion*. The settings include:
+
+- The Artemis base url. Can be changed to switch to a specific Artemis instance. Defaults to https://artemis.ase.in.tum.de. **Important:** The url must not end with a /, otherwise it will not work!
+- The Artemis exercise paths. Orion will suggest to store newly downloaded exercises at *default path/course/exercise name*, with the default path dependant of the setting.
+- The default commit message
+- An option to change the user agent if needed as well as a button to restart the integrated browser. If Orion gets stuck restarting the browser might solve the issue without restarting IntelliJ.
+
+Problems
+^^^^^^^^
+
+Please report any issues on `GitHub <https://github.com/ls1intum/Orion>`_.
+
+Orion for Students
+------------------
+
+Students can participate in programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the exercise using the integrated browser.
+ 2. After starting the exercise, a button to |open-in-intellij-button| appears.
+ 3. After cloning, the exercise can be solved in IntelliJ.
+ 4. Orion provides a button to |submit-button|. This will automatically commit and push the local changes and display the result of the automatic tests in IntelliJ.
+
+`Gif showcasing the usage for students <https://github.com/ls1intum/Orion#example-usage>`_
+
+Orion for Instructors
+---------------------
+
+Instructors can set up programming exercises via Orion by performing the following steps:
+
+ 1. The exercise needs to be created as described at the programming exercise creation step 1 and 2.
+ 2. After the creation, navigate to the instructor exercise overview using the integrated browser
+ 3. Each programming exercise will provide a button to |edit-in-intellij-button|. The button is rightmost in the table an might require scrolling. This will download the template, solution and test repository of the exercise
+ 4. Edit the repository files in IntelliJ.
+ 5. Orion provides a button to |submit-button|. This will commit and push all local changes to their respective repository.
```

Comment:
For me, it triggers the test locally button as well. Is this only a graphical bug (we had already such issues with other buttons) or a functional issue?

Reply:
Both buttons get put into loading mode since you are not supposed to trigger two actions in parallel. It would suprise me if the test was actually executed